runtime/jcl/common/shared.c

```diff
@@ -2314,7 +2314,8 @@ populateSharedCacheInfo(J9JavaVM *vm, J9SharedCacheInfo *event_data, void *user_
 												(-1 == event_data->cacheSize) ? (jlong) -1 : (jlong) event_data->cacheSize,
 												(-1 == event_data->freeBytes) ? (jlong) -1 : (jlong) event_data->freeBytes,
 												(jint)event_data->cacheType,
-												((UDATA)-1 == event_data->softMaxBytes) ? (jlong) -1 : (jlong) event_data->softMaxBytes
+												((UDATA)-1 == event_data->softMaxBytes) ? (jlong) -1 : (jlong) event_data->softMaxBytes,
+												(-1 == event_data->layer) ? (jint) -1 : (jint) event_data->layer
```

Comment:
I'm not sure why this (and likewise above) isn't just `(jint) event_data->layer`